Gemfile

```diff
@@ -44,6 +44,7 @@ end
 group :development do
   gem 'listen'
   gem 'partially_useful'
+  gem 'pry-rails', group: :development
```

Comment:
I prefer byebug, which is already included, but no hard feeling against this.

Reply:
I personally don't mind either of those two.
I changed/added it mainly because I had trouble getting the event model working in may rails console because the `Whitelabel.label` was not setup. Since a `.pryrc` is already present I felt easier to make the switch to `pry` instead of taking a close look with the `irb` was not working.

Alternatively, I could also just remove the [commit](https://github.com/phoet/on_ruby/pull/702/commits/4262665c878d6d643cd31031493e7c16a4f9c458) again since it technically has also nothing to do with the feature itself.

Reply:
I would like to ship the MR sooner than later.
Therefore I would close this conversation by the end of the day since this commit doesn't hurt in the end.

Reply:
Well, I approved it already, so it's on you to decide if you would like to change this or not.